src/components/FlatButton.tsx

```diff
@@ -1,18 +1,24 @@
 import React, { SFC } from 'react';
-import { StyleSheet } from 'react-native';
+import { StyleSheet, TouchableOpacity } from 'react-native';
 import { Button as ElementsButton, ButtonProps } from 'react-native-elements';
 
 import { palette, typography } from 'styles';
 
-export const FlatButton: SFC<ButtonProps> = props => (
-  <ElementsButton
-    {...props}
-    loadingProps={{ color: palette.primary }}
-    buttonStyle={[styles.button, props.buttonStyle]}
-    titleStyle={[styles.title, props.titleStyle]}
-    disabledStyle={styles.buttonDisabled}
-    disabledTitleStyle={styles.titleDisabled}
-  />
+interface Props extends ButtonProps {
+  handlePress?: () => void;
```

Comment:
`FlatButton` already has the `onPress` handler. Do we need another abstraction layer?